spec/features/users_index_spec.rb file

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  before :each do
+    @user1 = User.create!(name: 'Newton', photo: 'https://unsplash.com/photos/1.jpg', bio: 'Teacher from Mexico.',
```

Comment:
- [ ] Dear Team, this is not a valid spec file the `file` text comes after the file name. this makes it a regular text file. Remember each file is identified by its name and the file extension which comes after the `.`. please update the file name and make sure the tests are passing.